core/src/player.rs

```diff
@@ -1569,22 +1564,18 @@ impl Player {
         })
     }
 
-    /// Loads font data from the given buffer.
-    /// The buffer should be the `DefineFont3` info for the tag.
-    /// The tag header should not be included.
     pub fn load_device_font<'gc>(
         gc_context: gc_arena::MutationContext<'gc, '_>,
-        data: &[u8],
         renderer: &mut dyn RenderBackend,
-    ) -> Result<crate::font::Font<'gc>, swf::error::Error> {
-        let mut reader = swf::read::Reader::new(data, 8);
-        let device_font = crate::font::Font::from_swf_tag(
+    ) -> Font<'gc> {
+        const DEVICE_FONT_TAG: &[u8] = include_bytes!("../assets/noto-sans-definefont3.bin");
+        let mut reader = swf::read::Reader::new(DEVICE_FONT_TAG, 8);
+        Font::from_swf_tag(
             gc_context,
             renderer,
-            reader.read_define_font_2(3)?,
+            reader.read_define_font_2(3).unwrap(),
```

Comment:
Can we change this to `expect("Built-in font should compile")`? I want to move us away from unwraps.